concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1508,117 +1508,7 @@ jobs:
-                    6634e01b86fd7af541c19288ed9dd069126d9c9e1f508a63662dbb9781f4bae2
-
-                ./paas-cf/concourse/scripts/upload-buildpack.sh \
-                    go_buildpack cflinuxfs3 \
-                    go-buildpack-cflinuxfs3-v1.8.28.zip \
-                    https://github.com/cloudfoundry/go-buildpack/releases/download/v1.8.28/go-buildpack-cflinuxfs3-v1.8.28.zip \
-                    a344c51df41ce0ce0e411d2b981bbf3395b1a848cd3ce8b135e4648e3c50ab7d
-
-                ./paas-cf/concourse/scripts/upload-buildpack.sh \
-                    java_buildpack cflinuxfs3 \
-                    java-buildpack-cflinuxfs3-v4.16.1.zip \
-                    https://github.com/cloudfoundry/java-buildpack/releases/download/v4.16.1/java-buildpack-v4.16.1.zip \
-                    b02b6fe9f71fbdcc060e81860723a752a5e455dd63b37ca3d41caa63c130b563
-
-                ./paas-cf/concourse/scripts/upload-buildpack.sh \
-                    nodejs_buildpack cflinuxfs3 \
-                    nodejs-buildpack-cflinuxfs3-v1.6.32.zip \
-                    https://github.com/cloudfoundry/nodejs-buildpack/releases/download/v1.6.32/nodejs-buildpack-cflinuxfs3-v1.6.32.zip \
-                    dfd67dbde6e4c1330fdac773302cc2bd12c07085f6d0354ee8d4b73ac8558c5e
-
-                ./paas-cf/concourse/scripts/upload-buildpack.sh \
-                    php_buildpack cflinuxfs3 \
-                    php-buildpack-cflinuxfs3-v4.3.61.zip \
-                    https://github.com/cloudfoundry/php-buildpack/releases/download/v4.3.61/php-buildpack-cflinuxfs3-v4.3.61.zip \
-                    51f0b636067106badd30f9f14d6d0b049b299e7befe7429aa1ced6aa2792bfe0
-
-                ./paas-cf/concourse/scripts/upload-buildpack.sh \
-                    python_buildpack cflinuxfs3 \
-                    python-buildpack-cflinuxfs3-v1.6.21.zip \
-                    https://github.com/cloudfoundry/python-buildpack/releases/download/v1.6.21/python-buildpack-cflinuxfs3-v1.6.21.zip \
-                    c7c5f464c8a918b7c5b12b33fd38bf6f2cdd7d94602a42b0e70cba061f68e5ea
-
-                ./paas-cf/concourse/scripts/upload-buildpack.sh \
-                    ruby_buildpack cflinuxfs3 \
-                    ruby-buildpack-cflinuxfs3-v1.7.27.zip \
-                    https://github.com/cloudfoundry/ruby-buildpack/releases/download/v1.7.27/ruby-buildpack-cflinuxfs3-v1.7.27.zip \
-                    f934d2008543c6fa8a6ca471c71ef3d176d4ec598a4f7ded3913c4b52f96108a
-
-                ./paas-cf/concourse/scripts/upload-buildpack.sh \
-                    staticfile_buildpack cflinuxfs3 \
-                    staticfile-buildpack-cflinuxfs3-v1.4.32.zip \
-                    https://github.com/cloudfoundry/staticfile-buildpack/releases/download/v1.4.32/staticfile-buildpack-cflinuxfs3-v1.4.32.zip \
-                    7473873333aa4c18ffe4f92cce9ad270770881563a91dd7d0c4e13b8ea20efc7
-
+                ruby -ryaml ./paas-cf/concourse/scripts/parse_buildpacks_and_run_uploader.rb
```

Comment:
We shouldn't need to do the `-ryaml` here, instead the script should require what it needs.  Also, if we give the script a `#!` line and make it executable we won't need the `ruby` call here either, which will make it consistent with other ruby scripts.

Reply:
the key is named the same way as in the [buildpack manifest](https://github.com/cloudfoundry/python-buildpack/blob/master/manifest.yml).

Issue with removing it is that the first email we generate won't have any dependancy change information, unless we go off-process this time?

agreed on the ruby script - will modify that.